core/src/main/scala/kafka/tools/ProducerPerformance.scala

```diff
@@ -40,8 +40,10 @@ import org.apache.log4j.Logger
 object ProducerPerformance extends Logging {
```

Comment:
Not sure if I asked this before. Why is the help doc of this tool different from the other ones?

Reply:
So the command line uses the class ProducerPerformance.java under tools/src/main/java/org/apache/kafka/tools (which uses ArgumentParser) instead of this scala class(which uses JOptParser). I still just went ahead and made changes here to make it consistent with changes made in other scala classes.